api/backend/articles/routes.py

```diff
@@ -0,0 +1,69 @@
+from .. import api, db
+from ..articles.decorators import article_exists, valid_article_form
+from ..articles.utils import create_article, update_article
+from ..articles.schemas import article_schema, articles_schema 
+from ..models import Article
+from flask import Blueprint, request
+from flask_restful import Resource
+
+# Blueprint for articles
+articles_bp = Blueprint("articles", __name__)
+
+
+# Class to handle information regarding the Article model
+class ArticleCreate(Resource):
+    
+    @valid_article_form
+    def post(self):
+        form_data = request.get_json()
+        article = create_article(form_data)
+
+        db.session.add(article)
+        db.session.commit()
+
+        return {"message": "Successfully created Article"}, 201
+
+    # Display all the articles up to a limit of 15
+    def get(self):
+
+        #Check if the Article table has articles before printing upto 15 articles
+        if Article.query.first():
```

Comment:
We make this check into a decorator since that would make the routes file less crowded